docs/en/ingest-management/release-notes/release-notes-7.17.asciidoc

```diff
@@ -33,6 +35,48 @@ Also see:
 * {kibana-ref}/release-notes.html[{kib} release notes]
 * {beats-ref}/release-notes.html[{beats} release notes]
 
+// begin 7.17.7 relnotes
+
+[[release-notes-7.17.7]]
+== {fleet} and {agent} 7.17.7
+
+Review important information about the {fleet} and {agent} 7.17.7 release.
+
+[discrete]
+[[breaking-changes-7.17.7]]
+=== Breaking changes
+
+Breaking changes can prevent your application from optimal operation and
+performance. Before you upgrade, review the breaking changes, then mitigate the
+impact to your application.
+
+[discrete]
+[[breaking-PR32493]]
+.{fleet-server} now rejects certificates signed with SHA-1
```

Comment:
This applies to agent as well as Fleet server.

Reply:
Actually it appears we did not backport this change to 7.17 in fleet-server, so this only applies to agent in 7.17.